ui/Dockerfile

```
@@ -4,18 +4,13 @@ COPY contracts/package.json contracts/package-lock.json ./contracts/truffle-conf
 COPY ./contracts/contracts ./contracts
 RUN npm install --only=prod && npm run compile
```

We can do this to benefit from Docker caching mechanism:
First, remove `./contracts/truffle-config.js` from the line
```
COPY contracts/package.json contracts/package-lock.json ./contracts/truffle-config.js ./
```
Then do this:
```suggestion
RUN npm install --only=prod
COPY ./contracts/contracts ./contracts
RUN npm run compile
```

---

Very smart thanks

---

```suggestion
RUN npm install --only=prod 
COPY ./contracts/truffle-config.js ./
COPY ./contracts/contracts ./contracts
RUN npm run compile
```
Would it be the same here ? because `truffle-config.js` needs to be on the root ?

---

Yes I think that would be correct.